coalib/processes/Processing.py

```diff
@@ -461,6 +461,31 @@ def yield_ignore_ranges(file_dict):
                                line_number, 1,
                                end_line, len(file[end_line - 1])))
 
+            # Before lowering all lines ever read, first look for the biggest
+            # common substring, case sensitive: N*oqa, start n*oqa.
```

Comment:
hey, you copied the code above right? That's code duplication.That's usually not good because if we fund a bug in the code above it'll probably be forgotten to fix in your new code. Usually it's way easier to maintain if you don't duplicate chunks of code. Rather put them in a function, give it a variable or make a loop or something. I think we can be smart about it and make this without adding any lines of code by modifying the conditions